packages/core/src/types.ts

```diff
@@ -102,6 +103,7 @@ export interface Options extends GetPluginsOptions {
     [rum.CONFIG_KEY]?: RumOptions;
     [telemetry.CONFIG_KEY]?: TelemetryOptions;
     // #types-injection-marker
+    customPlugins?: GetCustomPlugins<Options>[];
```

Comment:
I don't know if accepting an array of functions with each of them returning an array of plugins is necessary 🤔 
IMO having a single function returning an array of plugins should be enough, is there a particular reason to not do so?

```diff
// This is what I mean:
- customPlugins?: GetCustomPlugins<Options>[];
+ customPlugins?: GetCustomPlugins<Options>
```

Reply:
I thought about it too, but I didn't go into it deeply enough.
So no strong opinion on it at all, and I agree with you that a single function returning an array would be plenty enough.

I'll updated.